release_dashboard/test/logic/error_to_string_test.dart

```diff
@@ -19,21 +19,23 @@ void main() {
         errorOutput = errorToString(err, stackTrace);
       }
 
-      expect(errorOutput.contains('Conductor Exception:'), true);
+      expect(errorOutput.contains('ConductorException:'), true);
       expect(errorOutput.contains(errorMsg), true);
       expect(errorOutput.contains('Stack Trace:'), true);
       expect(errorOutput.contains(stackTraceMsg), true);
     });
 
     test('errorToString displays the correct general exception with its stack trace', () async {
       late final String errorOutput;
+      late final String runtimeType;
       try {
```

Comment:
Instead of a try/catch, you can pass Exception directly to errorToString.

Reply:
Ok, I removed try/catch for the first test. But for the second tests, I need the `'${err.runtimeType}'` from the catch.